Mondee Watch App/View/Guide/GuideView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  GuideView.swift
+//  Mondee Watch App
+//
+//  Created by Seokmin on 2023/07/15.
+//
+
+import SwiftUI
+
+struct GuideView: View {
+    @Binding var gameStatus: GameStatus
+    @State var countDownView = false
```

Comment:
영의 코드리뷰시간에서 알려주신 것인데 다른종류의 변수는 띄어쓰기가 좋다고 하셨습니다! 
그리고 View 타입인 CountDownView와 불값인 countDownView 변수가 이름이 똑같아서 헷갈립니다! 변수이름에 active가 들어가면 좋을 것 같습니다!
```suggestion
    @Binding var gameStatus: GameStatus
    
    @State var countDownView = false
```